### PR TITLE
Add try/catch blocks with known issue link to the factory tests

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Add try/catch blocks with known issue link to the factory tests which described in the issue https://github.com/eclipse/che/issues/7940

### What issues does this PR fix or reference?
Issue https://github.com/eclipse/che/issues/7940

### CODE REVIEW

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java [hunk 1]
```diff
@@ -147,9 +147,15 @@ private void createFactoryAndSwitchToWs() throws IOException {
...
-    projectExplorer.expandPathInProjectExplorerAndOpenFile(
-        PROJECT_NAME + "/" + KEEPED_DIR + "/src" + "/main" + "/java/hello",
-        "GreetingController.java");
+    try {
+      projectExplorer.expandPathInProjectExplorerAndOpenFile(
+          PROJECT_NAME + "/" + KEEPED_DIR + "/src" + "/main" + "/java/hello",
+          "GreetingController.java");
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7959");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java:156]: It's better to pass "_ex_" variable into the **fail()** method to have additional info about the error in the test log.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java [hunk 2]
```diff
@@ -94,7 +96,14 @@ public void createFactoryFromDashBoard() throws ExecutionException, InterruptedE
     events.waitExpectedMessage(CONFIGURING_PROJECT_AND_CLONING_SOURCE_CODE);
     events.waitExpectedMessage("Project " + PROJECT_NAME + " imported");
     notificationsPopupPanel.waitPopUpPanelsIsClosed();
-    projectExplorer.openItemByPath(PROJECT_NAME);
+
+    try {
+      projectExplorer.openItemByPath(PROJECT_NAME);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7959");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java:104]: It's better to pass "_ex_" variable into the **fail()** method to have additional info about the error in the test log.

**Review by tolusha — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java (hunk 1 above)
Comment (skabashnyuk) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java:156]: Please keep the output of the log short. I think a message is enough. No need to put giant stack trace to output.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java (hunk 2 above)
Comment (skabashnyuk) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java:104]: Please keep the output of the log short. I think a message is enough. No need to put giant stack trace to output.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java (hunk 1 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java:156]: okay

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java (hunk 2 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateNamedFactoryFromDashBoard.java:104]: okay

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**